.github/workflows/cronjobs.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
     uses: securesign/actions/.github/workflows/check-image-version.yaml@main
     strategy:
       matrix:
-        branch: [main, midstream-v1.2.2]
+        branch: [redhat-v1.2.2, redhat-v1.3]
```

Comment:
Can you also keep `main` here? I think we'll eventually want it anyway. And I don't think we need `redhat-v1.2.2` since we're not going to be maintaining that branch for the long run.